src/test/resources/BulkCopyCSVSendTemporalDataTypesAsStringForBulkCopy2.csv

```diff
@@ -0,0 +1,2 @@
+int,date,datetime,datetime2,time,datetimeoffset,smalldatetime,money,smallmoney
+1,0001-01-01,1753-01-01 00:00:00,0001-01-01 00:00:00,00:00:00,2025-12-10 12:32:10,1900-01-01 06:56:00,100.2523,100.2523
```

Comment:
New line here.

Reply:
will add.

Reply:
Actually, I don't think we should add a newline here, because it's a CSV file. AFAIK if a CSV file only had 1 column and the file has a newline at the end, I think that would be treated as a NULL value, which might not always be what's being tested (I remember specifically not leaving a newline at the end of CSV files for this reason)